crates/autopilot/src/domain/settlement/trade.rs

```diff
@@ -39,102 +43,491 @@ impl Trade {
         }
     }
 
+    /// CIP38 score defined as surplus + protocol fee
+    ///
+    /// Denominated in NATIVE token
+    #[allow(dead_code)]
+    fn score(
```

Comment:
why was this function moved here?

Reply:
If you refer to driver, it's not actually moved but reimplemented because we want autopilot to be able to calculate the score as well.